content/doc/book/blueocean/dashboard.adoc

```diff
@@ -19,105 +19,78 @@ endif::[]
 
 = Dashboard
 
-Blue Ocean's "Dashboard" is the default view shown when you open Blue Ocean and
-shows an overview of all Pipeline projects configured on a Jenkins instance.
+Blue Ocean's *Dashboard* is the default view shown when opening Blue Ocean.
+It disaplays an overview of all Pipeline projects configured on a Jenkins instance.
 
 include::doc/book/blueocean/_blue-ocean-status.adoc[]
 
-The Dashboard consists of a blue link:#navigation-bar[navigation bar] at the
-top, the link:#pipelines-list[Pipelines list], as well as the
-link:#favorites-list[Favorites list].
+The Dashboard consists of a blue link:#navigation-bar[navigation bar] at the top, the link:#pipelines-list[Pipelines list], and the link:#favorites-list[Favorites list].
 
 [.boxshadow]
-image:blueocean/dashboard/overview.png[Overview of the
-Dashboard,role=center,width=100%]
+image:blueocean/dashboard/overview.png[Overview of the Dashboard,role=center,width=100%]
 
 == Navigation bar
 
-The Dashboard includes the blue-colored
-link:../getting-started#navigation-bar[navigation bar] along the top of the
-interface.
+The Dashboard includes the blue link:../getting-started#navigation-bar[navigation bar] along the top of the interface.
 
-This bar is divided into two sections - a common section along the top and a
-contextual section below. The contextual section changes depending on the
-current Blue Ocean page you are viewing.
+This bar is divided into two sections:
 
-When viewing the Dashboard, the navigation bar's contextual section includes
-the:
+* A common section along the top 
+* A contextual section below
```

Comment:
```suggestion
* A common section along the top.
* A contextual section below.
```

Reply:
Good callout, I missed this during my proofreading, thanks @cari-mason !